lessons/students/marking_period_1/week_2/8_error_handling.py

```diff
@@ -2,15 +2,35 @@
 Common errors you will/may have already encounter
```

Comment:
This doesn't seem right to me. It should either be:
- Common errors you will encounter.
or maybe
- Common errors you will/may have already encounter(ed).